ui/apps/platform/src/utils/chartUtils.ts

```diff
@@ -0,0 +1,56 @@
+import { History } from 'react-router-dom';
+import { getTheme, ChartThemeColor } from '@patternfly/react-charts';
+import { EventCallbackInterface, EventPropTypeInterface } from 'victory-core';
+
+const severityColorScale = [
+    'var(--pf-chart-color-black-300)',
+    'var(--pf-chart-color-gold-300)',
+    'var(--pf-chart-color-orange-300)',
+    'var(--pf-chart-color-red-100)',
+];
```

Comment:
Let's add a TODO to lift these colors into a global constant list in our app.

cc: @alanonthegit

Reply:
> @vjwilson Related to your comment about the `@patternfly/patternfly` dependency. How do you feel about dropping the dependency and using the colors that are part of PF's global palette instead? I can see us reusing these colors in the future in contexts that aren't chart specific, and it would save us some kbs in the bundle.
> 
> The global palette matches the chart colors exactly, except for the gray, which is -1 red and +1 blue component compared to the charts version (pictured on the right below).
> 
> <img alt="image" width="964" src="https://user-images.githubusercontent.com/1292638/172676747-d08e54ac-3f14-4c81-988b-fb6ae9efcbe7.png">

Ok with me. What do you think @alanonthegit  ?